lib/bundler/templates/newgem/bin/setup.tt

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
+cd -
```

Comment:
Yes, it was my mistake.

Please, check the following commit, I've wrote a proper version of cd.